Deployment/asdk-installer.ps1

```diff
@@ -1643,12 +1646,15 @@ Function F_Summary {
         $InstallScript += "`r`n"
         $InstallScript += '.\InstallAzureStackPOC.ps1 -AdminPassword $adminpass'
 
-        If ($synchash.Control_Creds_Cbx_Idp.SelectedItem -eq 'Azure Cloud') {
```

Comment:
For easier readability and the option to change parameters I suggest to do one if else tree instead of nested conditions.

If ($synchash.Control_Creds_Cbx_Idp.SelectedItem -eq 'Azure Cloud') {
        $InstallScript += " -InfraAzureDirectoryTenantName "
        $InstallScript += $synchash.Control_Creds_Tbx_AADTenant.Text
}

ElseIf ($synchash.Control_Creds_Cbx_Idp.SelectedItem -eq 'Azure China Cloud') {
        $InstallScript += " -InfraAzureDirectoryTenantName "
        $InstallScript += $synchash.Control_Creds_Tbx_AADTenant.Text
        $InstallScript += " -InfraAzureEnvironment AzureChinaCloud" 
}

ElseIf ($synchash.Control_Creds_Cbx_Idp.SelectedItem -eq 'ADFS') {
        $InstallScript += " -UseADFS"
}